release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -2029,6 +2037,23 @@ In the following tables, features are marked with the following statuses:
 
 * A regression in the behaviour of `libreswan` caused some nodes with IPsec enabled to lose communication with pods on other nodes in the same cluster. To resolve this issue, consider disabling IPsec for your cluster. (link:https://issues.redhat.com/browse/OCPBUGS-43713[*OCPBUGS-43713*])
 
+* There is a known issue in {product-title} version 4.18 that prevents configuring multiple subnets in the failure domain of a Nutanix cluster during installation.
+There is no workaround for this issue.
+(link:https://issues.redhat.com/browse/OCPBUGS-49885[*OCPBUGS-49885*])
+
+ * The following known issues with configuring multiple subnets for an existing Nutanix cluster by using a control plane machine set exist in {product-title} version 4.18:
```

Comment:
This is a little bit of a mouthful. Is there any way to make this a little more concise/clearer? I think that "exist" being so far away is what's throwing me off. I also don't think it's super important to mention OCP 4.18 here since this is in the 4.18 RNs. Suggestion below

```suggestion
 * The following known issues exist with configuring multiple subnets for an existing Nutanix cluster by using a control plane machine set:
```

Reply:
Haha - that's an artifact of this originating in the main docs. Good observation, I'll trim this version.